### DISCUSSION TITLE
Looking for the working code for cisco_iosxe with asyncio

### DISCUSSION THREAD

**Discussion by HaiderNL** [scrapli/scrapli_community#154] [Q&A] [answered]
Where is the latest code for cisco iosxe platforms with asyncio ?
Just "send_command" task is fine. A single device is fine too.
I'll take the code to the next level by myself... Just looking for the base code.

**Comment by carlmontanari** [comment, marked as answer]
[here](https://github.com/carlmontanari/scrapli/tree/main/examples/async_usage)
